atlascope/core/rest/endpoints/investigation_endpoints.py

```diff
@@ -38,3 +38,18 @@ def jobs(self, request, pk=None):
     def embeddings(self, request, pk=None):
         payload = DatasetEmbeddingSerializer(self.get_object().embeddings.all(), many=True).data
         return Response(payload, status=status.HTTP_200_OK)
+
+    # @swagger_auto_schema(request_body=TourCreateSerializer())
+    # @action(detail=True, methods=['POST'])
+    # def tour(self, request, pk=None):
+    #     serializer = TourCreateSerializer(data=request.data)
```

Comment:
Why did you decide to comment this out?

Reply:
If this stuff isn't needed for the current PR, let's cut it out.

Reply:
Commented create stuff removed and read endpoint added in 19ba0032d9cea5ca607ae52bea612feec0d07ebd